docs/howto/team-open-science-mindset.md

```diff
@@ -21,46 +21,77 @@ We want our community ontology editors and developers to be _excited_ to be part
 
 1. **Take ownership of our community ontologies**
 
-- Ensure that you see your issues and Pull Requests through to the end
-- Feel empowered to nag reviewers or experts to help
+    - Ensure that you see your issues and Pull Requests through to the end
+    - Feel empowered to nag reviewers or experts to help
+
+Example: After waiting for the PR to be reviewed, Meghan kindly asked Nicole if she should find a different reviewer.
+
+<img width="787" alt="image" src="https://user-images.githubusercontent.com/6722114/190458254-ed13ef7b-9c0a-455a-ad49-89d2548b81ab.png">
+
 
 1. **Reduce the fear of breaking the ontology**
 
-- We've all probably already made the same mistakes before - it's okay to break things or or make mistakes..
-- We have so many checks in place and GitHub has version control. Nothing has ever been broken to the point where it can't be fixed.
-- Perfect is the enemy of good enough.
-- It's okay if your PR is rejected
+    - We've all probably already made the same mistakes before - it's okay to break things or or make mistakes.
+
+Example: The QC checks on [this PR](https://github.com/monarch-initiative/mondo/pull/5266) failed 6 times before it passed.
+
+<img width="788" alt="image" src="https://user-images.githubusercontent.com/6722114/190459249-e52957a6-8d8f-4ba8-bd7d-547039a7c76d.png">
+
+    - We have so many checks in place and GitHub has version control. Nothing has ever been broken to the point where it can't be fixed. (Remember to work on a branch!)
+    - Perfect is the enemy of good enough.
+    - It's okay if your PR is rejected.
 
 1. **Be proactive**
 
-- Problem of decentralization and lack of hierarchy needs proactive and brave decision makers
-- No one will do your work for you
-- Learn the tools using OBOOK, make fixes - just try it/do it
-- Can improve the materials by fixing it
+    - Problem of decentralization and lack of hierarchy needs proactive and brave decision makers.
+    - No one will do your work for you.
+    - Learn the tools using OBOOK, make fixes - just try it/do it.
+    - Can improve the materials by fixing it (please feel empowered to fix anything, including typos and content).
+
+<img width="1080" alt="image" src="https://user-images.githubusercontent.com/6722114/190459617-ba48bafd-e4ef-44e2-ac6e-d70de1cd8147.png">
 
 1. **Be an open scientist**
 
-- Contribute things openly so other people can benefit from it
-- ie don't put issues on Slack where the public can't see it.
-- Create public tickets
+    - Contribute things openly so other people can benefit from it
```

Comment:
Inconsistent use of full stops at the end of bulleted points. Consider adding full stops to all.

Reply:
Will deal with this later